packages/react-components/react-toolbar/src/components/Toolbar/Toolbar.types.ts

```diff
@@ -34,12 +34,12 @@ export type ToolbarProps = ComponentProps<ToolbarSlots> & {
   /**
    * Map of all checked values
    */
-  checkedValues?: Record<string, string[]>;
+  checkedValues?: Record<string, string[] | string>;
 
   /**
    * Default values to be checked on mount
    */
-  defaultCheckedValues?: Record<string, string[]>;
+  defaultCheckedValues?: Record<string, string[] | string>;
```

Comment:
I think in this case it's still to return an array, just document that for radio items the array will only have one value

Reply:
So, in all cases, should it be always array? I was questioning myself when doing it, so happy to change it.

Reply:
Done https://github.com/microsoft/fluentui/pull/25343/commits/cfd0b822ed8a44f7f6c6e75c60e20ef929cb9a72